1747998762383	Sync.LogManager	DEBUG	Flushing file log
1747998762384	Sync.LogManager	DEBUG	Log cleanup threshold time: 1747134762384
1747998762385	Sync.LogManager	DEBUG	Done deleting files.
1747998768021	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1747998772944	Sync.SyncScheduler	DEBUG	Network link status changed to "down". Offline?: false
1747998772944	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1747998772945	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=schedule)
1747998772947	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1747998775270	Sync.SyncScheduler	DEBUG	Network link status changed to "up". Offline?: false
1747998775270	Sync.SyncScheduler	DEBUG	Network link looks up. Syncing.
1747998775273	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1747998775274	Sync.Service	DEBUG	User-Agent: Firefox/138.0.4 (Linux x86_64) FxSync/1.140.0.20250518015817.desktop
1747998775274	Sync.Service	INFO	Starting sync at 2025-05-23 16:42:55 in browser session wt9PXLXT_ZJY
1747998775274	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1747998775276	Sync.Status	INFO	Resetting Status.
1747998775276	Sync.Status	DEBUG	Status.service: error.sync.failed => success.status_ok
1747998775277	FirefoxAccounts	TRACE	not checking freshness of profile as it remains recent
1747998778162	Sync.SyncScheduler	DEBUG	Network link status changed to "down". Offline?: false
1747998778163	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1747998778163	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=schedule)
1747998778165	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1747998780336	Sync.Resource	WARN	GET request to https://sync-1-us-west1-g.sync.services.mozilla.com/1.5/195504406/info/collections failed: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1747998780336	Sync.Status	DEBUG	Status.sync: success.sync => error.login.reason.network
1747998780336	Sync.Status	DEBUG	Status.service: success.status_ok => error.sync.failed
1747998780337	Sync.ErrorHandler	ERROR	Sync encountered an error: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1747998780338	Sync.SyncScheduler	DEBUG	Sync error count has exceeded 3; enforcing backoff.
1747998780338	Sync.SyncScheduler	DEBUG	Starting client-initiated backoff. Next sync in 3869718 ms.
1747998780338	Sync.SyncScheduler	DEBUG	Next sync in 3869718 ms. (why=client-backoff-schedule)
1747998780341	Sync.Service	DEBUG	Exception calling anonymous function: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
